extensions/vro/cors/impl/pom.xml

```diff
@@ -46,6 +46,31 @@
 		    <artifactId>spring-web</artifactId>
 		</dependency>
 
+        <!-- TESTS -->
+        <dependency>
+            <groupId>org.apache.isis.core</groupId>
+            <artifactId>isis-core-internaltestsupport</artifactId>
+            <scope>test</scope>
+            <!-- provided by Spring Boot -->
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-api</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>javax.servlet-api</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.tomcat.embed</groupId>
+			<artifactId>tomcat-embed-core</artifactId>
+			<version>9.0.37</version>
+			<scope>test</scope>
+		</dependency>
+
```

Comment:
Might be opinionated, but lets rather not bring in this heavy lifting test infrastructure into these tiny extension modules. That's what the regressiontests module is for.